lab_1/main.py

```diff
@@ -51,9 +73,14 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
+    if isinstance(language, str) and isinstance(text, str) and isinstance(stop_words, list):
```

Comment:
There should be a check for the input arguments at the top of the function - after the documentation string. Remove nesting here and extract the check into a separate `if` block earlier.